src/main/java/com/yana/internship/entity/Order.java

```diff
@@ -0,0 +1,83 @@
+package com.yana.internship.entity;
+
+import org.hibernate.annotations.CreationTimestamp;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Entity
+@Table(name = "app_order")
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
+
+    @CreationTimestamp
+    @Temporal(TemporalType.TIMESTAMP)
```

Comment:
Let's use common data format for all fields. Date or Timestamp

Reply:
Changed to Date